libs/sdk-backend-tiger/src/backend/workspace/insights/index.ts

```diff
@@ -62,9 +66,7 @@ export class TigerWorkspaceInsights implements IWorkspaceInsightsService {
     };
 
     public getInsights = async (options?: IInsightsQueryOptions): Promise<IInsightsQueryResult> => {
-        const {
-            data: { data: visualizationObjects, links, meta },
-        } = await this.authCall((sdk) => {
+        const resultPromise = await this.authCall((sdk) => {
```

Comment:
Naming: since you `await` the variable no longer represents a promise, so the name is misleading. What about just `result` or `response`?